pkg/controllers/disruption/controller.go

```diff
@@ -114,6 +114,9 @@ func (c *Controller) Reconcile(ctx context.Context, _ reconcile.Request) (reconc
 	// with making any scheduling decision off of our state nodes. Otherwise, we have the potential to make
 	// a scheduling decision based on a smaller subset of nodes in our cluster state than actually exist.
 	if !c.cluster.Synced(ctx) {
+		if time.Since(c.cluster.LastSyncTime()) > state.AbnormalClusterSyncDuration {
+			logging.FromContext(ctx).Infof("cluster state has not been in sync since %v, waiting for sync", c.cluster.LastSyncTime())
+		}
```

Comment:
This is interesting, since you'd have no logs for 15 seconds, but then spam it continuously after.

Additionally, I'd name the log to match the variable if you're naming it like this
```suggestion
		if time.Since(c.cluster.LastSyncTime()) > state.AbnormalClusterSyncDuration {
			logging.FromContext(ctx).Infof("detected abnormal duration since cluster was last synced %v", c.cluster.LastSyncTime())
		}
```